ThirdPartyAdapters/liftoffmonetize/liftoffmonetize/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java

```diff
@@ -310,28 +312,13 @@ public void loadRewardedAd(
 
     Context context = mediationRewardedAdConfiguration.getContext();
 
-    VungleInitializer.getInstance()
```

Comment:
If you remove this, what about lazy loading where the init for the SDK is not called?

For iOS, we still call init, but in the SDK, it will instantly call the callback.

Reply:
yes same qn. on IOS we still call init before load.

Reply:
@ManojBudumuru-vungle SDK supported load ad without init, right? I tested by this case:
1. not call init vungle SDK.
2. load fullscreen and banner ads

Reply:
where did u get the app id?